ksql-engine/src/main/java/io/confluent/ksql/analyzer/Analysis.java

```diff
@@ -71,11 +71,16 @@ public boolean isDoCreateInto() {
     return doCreateInto;
   }
 
-
   public List<Pair<StructuredDataSource, String>> getFromDataSources() {
     return fromDataSources;
   }
 
+  public StructuredDataSource getTheSource() {
+    return (join == null)
```

Comment:
Storytime! Back at LinkedIn, I used to always omit the parenthesis for ternary operators. Then some senior developer came around and asked me to put clarifying parens on my ternary operators. I grew to like it since it made it easier to grok the condition and makes it easy to separate from assignments (e.g. `var x = (y == z) ? ... : ...`). But I don't care that much, so if you feel strongly that they clutter the code I'll remove them and stop in the future unless the condition is compound.

Reply:
I'd just stop in the future as its not something the code base current does.  Also, given we normally have ternary expressions over multiple lines its of limited benefit even for improving grokking.